README.md

```diff
@@ -3,15 +3,19 @@ Ballerina JWT Library
 
   [![Build](https://github.com/ballerina-platform/module-ballerina-jwt/actions/workflows/build-timestamped-master.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerina-jwt/actions/workflows/build-timestamped-master.yml)
   [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-jwt.svg?label=Last%20Commit)](https://github.com/ballerina-platform/module-ballerina-jwt/commits/master)
-  [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/jwt.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fjwt)
+  [![GitHub issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/jwt.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fjwt)
   [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
   [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerina-jwt/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerina-jwt)
 
-The JWT library is one of the standard library modules of the <a target="_blank" href="https://ballerina.io/">Ballerina</a> language.
+The JWT library is one of the standard library modules of the [Ballerina](https://ballerina.io/) language.
 
-It provides a listener JWT authentication provider, which can be used to authenticate the provided credentials against the provided JWT validator configurations, and a client self-signed JWT authentication provider, which can be used to authenticate against an external endpoint with a self-signed JWT issued against the provided JWT issuer configurations. Also, this module provides the functionality related to issuing and validating a JWT.
+This module provides a framework for authentication/authorization with JWTs and generation/validation of JWTs as specified in [RFC 7519](https://datatracker.ietf.org/doc/html/rfc7519), [RFC 7515](https://datatracker.ietf.org/doc/html/rfc7515) and [RFC 7517](https://datatracker.ietf.org/doc/html/rfc7517).
 
-For more information go to [The JWT Module](https://ballerina.io/learn/api-docs/ballerina/#/ballerina/jwt/latest/jwt/).
+JSON Web Token (JWT) is a compact, URL-safe means of representing claims to be transferred between two parties. The claims in a JWT are encoded as a JSON object that is used as the payload of a JSON Web Signature (JWS) structure or as the plaintext of a JSON Web Encryption (JWE) structure, enabling the claims to be digitally signed or integrity protected with a Message Authentication Code(MAC) and/or encrypted.
```

Comment:
Is it "claims" or "claim"? Seems it changes along with this description.

Reply:
It is "claims". WDYM by "Seems it changes along with this description."?

Reply:
Probably I misunderstood it :). Are `multiple` claims in a JWT encoded as a single JSON object? or `multiple` claims in a JWT encoded as `multiple` JSON objects.

Reply:
Ah yes. "multiple claims in a JWT encoded as a single JSON object" is the correct one. Actually this is the exact definition provided by the RFC :)